docs/rules/order.md

```diff
@@ -164,6 +163,26 @@ import index from './';
 import sibling from './foo';
 ```
 
+### `alphabetize`: boolean
+
+By default, no ordering within groups is enforced. When `alphabetize: true` is used, imports must be alphabetically ordered within their groups.
```

Comment:
This seems a bit restrictive; will people always want a-z? what about z-a? What about capital letters? Should it be `a, A, b, c, D`, or `A, a, b, c, D`, or `a, b, c, A, D`?

Reply:
Great questions. When I [took a run at this in the past](https://github.com/jfmengels/eslint-plugin-import-order/pull/12#issuecomment-210600202), I got some pushback around finding a balance between flexibility and simplicity.

Today's approach was more conservative, and the results aren't perfect. For example, we'd love to be able to differentiate webpack aliases from external node modules.

Basically, I want a completely stable sort, and alphabetical is pretty politically-neutral.
[localCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) accepts some options that it would be pretty straight-forward to thread through from the eslint config, if that's something users want.

Reply:
> Basically, I want a completely stable sort, and alphabetical is pretty politically-neutral.

I, as a person who's interested in using this lint, second this.

Reply:
@ljharb what's your thought on @duncanbeevers response to your comment here? I am thinking a-z ordering should be good enough for majority of users.

Reply:
I think a-z and z-a is a minimum, as well as an option that chooses between “z comes before/after A” and “z is right next to Z”. I definitely don’t want an arbitrary comparator.